packages/slice-machine/server/src/api/screenshots/generate.ts

```diff
@@ -21,53 +21,28 @@ export interface ScreenshotResults {
   failure: FailedScreenshot[];
 }
 
-export async function generateScreenshotAndRemoveCustom(
-  env: BackendEnvironment,
-  libraryName: string,
-  sliceName: string
-): Promise<ScreenshotResults> {
-  const { screenshots, failure } = await generateScreenshot(
-    env,
-    libraryName,
-    sliceName
-  );
-
-  // Remove custom screenshot of success
-  Object.keys(screenshots).forEach((variationId) =>
-    removeCustomScreenshot(env, libraryName, sliceName, variationId)
-  );
-
-  return {
-    screenshots: screenshots,
-    failure: failure,
-  };
-}
-
 export async function generateScreenshot(
```

Comment:
This will largely be refactored since we don't need the multiple promise logic anymore. Still WIP

Reply:
nice 👍, might not even need to be on a different file anymore.